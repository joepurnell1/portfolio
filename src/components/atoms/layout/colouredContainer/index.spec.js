import React from 'react';
import renderer from 'react-test-renderer';
import ColouredContainer from './index';

describe('coloured container component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <ColouredContainer>When Im watchin my TV and a man comes on and tells me</ColouredContainer>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('should with given colour as expected', () => {
    const render = renderer.create(
      <ColouredContainer backgroundColour="#907A70">How white my shirts can be</ColouredContainer>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
