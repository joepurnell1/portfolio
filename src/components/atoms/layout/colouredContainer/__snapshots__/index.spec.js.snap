// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`coloured container component should render default as expected 1`] = `
<div
  className="sc-bwzfXH uTxCW sc-bdVaJa gbwpMQ"
>
  When Im watchin my TV and a man comes on and tells me
</div>
`;

exports[`coloured container component should with given colour as expected 1`] = `
<div
  className="sc-bwzfXH uTxCW sc-bdVaJa fXjPLb"
>
  How white my shirts can be
</div>
`;
